js/components/portfolio/OperationsComponent.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * CITTADINANZA DIGITALE
+ * io.italia.it
```

Comment:
I'd not embed the brand (url) in comments

Reply:
Gotcha